apps/examples/src/examples/yjs/README.md

```diff
@@ -3,4 +3,5 @@ title: YJS
 component: ./YjsExample.tsx
 category: collaboration
 priority: 2
+keywords: [multiplayer]
```

Comment:
i might suggest making this two words "multi" and "player" just in case someone types it in like `multi-player`